astropy/cosmology/tests/test_flrw.py

```diff
@@ -717,6 +717,30 @@ def setup_class(self):
     # ===============================================================
     # Method & Attribute Tests
 
+    _FLRW_redshift_methods = [
```

Comment:
would probably be better to do this as ``{m for m in dir(FLRW) if callable(gettatr(FLRW, m)} - {the commented out stuff}``

Reply:
Done!